benchmarks/go_benchmark_test.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"errors"
```

Comment:
Shouldn't it be intended by 2 spaces?

Reply:
This is just tab, which seems appears to be 6 spaces on github.
I think this is the same as [other go file](https://github.com/golang/protobuf/blob/master/descriptor/descriptor.go)